src/codegen/resources/rest/auditlog/get_audit_log

```diff
@@ -0,0 +1,5 @@
+qs https://discordapp.com/developers/docs/resources/audit-log#get-guild-audit-log-query-string-parameters
+user_id?	snowflake	The id of the queried user.
+action_type?	integer	The type of the queried action.
+before?	snowflake	The snowflake of which to query before.
+limit?	integer	The limit of the audit logs received.
```

Comment:
We know these are all optional? they are not marked as such.

Reply:
I don't actually have access to `discordapp.com` however last time I checked, I sent an empty request with nothing and it worked.

Reply:
It makes sense that they're all optional. They just usually document it 🤷‍♀️

Reply:
`usually` is a good word there, there's quite a bit of undocumented stuff haha